tyler.py

```diff
@@ -16,29 +16,15 @@ def tyler(self, msg, args):  # a command callable with !tryme
         Feel free to tweak me to experiment with Errbot.
         You can find me in your init directory in the subdirectory plugins.
         """
```

Comment:
I think this is where we should place something that checks for an arg.

I think an arg like gif would be good, and would look something like:

`elif "gif" in arg:
    Your stuff`

You can see the most recent merge for examples.